src/main/java/com/playmonumenta/networkrelay/util/MMLog.java

```diff
@@ -0,0 +1,70 @@
+package com.playmonumenta.networkrelay.util;
+
+import com.playmonumenta.networkrelay.CustomLogger;
+import java.util.function.Supplier;
+import java.util.logging.Level;
+
+public class MMLog {
```

Comment:
I'm surprised we didn't add this to NetworkRelay years ago - it saves you from needing to get the custom logger instance all the time and check if it's valid. I'm also confused as to why `CustomLogger.getInstance()` returns an `Optional` instead of a `@Nullable`. None of that is your fault, and I see why it would make copying it frustrating.

Reply:
The Optional part was a me thing, there wasn't an getInstance method nor INSTANCE field before.
I just don't like null checks.